lib/decidim_app/k8s/organization_exporter.rb

```diff
@@ -15,7 +15,13 @@ class OrganizationExporter
                                       SCALEWAY_ID
                                       SCALEWAY_TOKEN
                                       SCALEWAY_BUCKET_NAME
-                                      SECRET_KEY_BASE).freeze
+                                      SECRET_KEY_BASE
+                                      RACK_ATTACK_ENABLED).freeze
```

Comment:
Change the env var to `ENABLE_RACK_ATTACK`
cf. https://github.com/OpenSourcePolitics/decidim-app/blob/fix/configuration-exporter-env-vars/config/secrets.yml#L18C1-L18C1